cmd/generator/main.go

```diff
@@ -119,7 +120,8 @@ var generatorCmd = &cobra.Command{
 
 			releasePlugins, err := getReleasePlugins(ctx, client, repositoryName, includePreRelease, existingPlugins)
 			if err != nil {
-				return errors.Wrapf(err, "failed to release plugin for repository %s", repositoryName)
+				logger.Warn(errors.Wrapf(err, "failed to get release plugins for repository %s", repositoryName))
```

Comment:
Actually, shouldn't we just error out here? If we fail to get release plugins for a repository, we might /remove/ information from the existing `plugins.json`, no?

Reply:
agreed.